SpatialGDK/Documentation/content/get-started/get-started.md

```diff
@@ -0,0 +1,24 @@
+# Get started: Introduction
+
+We recommend you start by following the [Installation and setup guide]({{urlRoot}}/setup-and-installing.md) to set up the GDK, its Unreal Engine fork, and our Starter Project. 
+
+At this stage, you’ll have the stack set up and be able to run your first SpatialOS deployments - both locally and in the cloud - to get an idea of what the platform provides.
+
+After this, check out the Multi-Server Shooter tutorial, where you can try out the GDK’s development experience by adding a new feature to the Starter Project and test it across two servers.
+
+Finally, if you have an existing Unreal multiplayer project, you can follow our detailed porting guide to get it onto the GDK!
+
+## Your feedback and ideas
+
+We'd love to hear from you - please don't hesitate to give us feedback on your onboarding experience, our documentation, our [roadmap](https://trello.com/b/7wtbtwmL/unreal-gdk-roadmap) (Trello board), or anything else!
```

Comment:
"Please don't hesitate" is a bit formal...
Suggest:

"We'd love to hear from you - drop into our forums or discord to give us feedback on your getting started experience, our documentation, our..."

Reply:
nice